app/models/topical_event.rb

```diff
@@ -1,8 +1,11 @@
 class TopicalEvent < ApplicationRecord
+  include DateValidation
   include PublishesToPublishingApi
   include Searchable
   include SimpleWorkflow
 
+  date_attributes(:start_date, :end_date)
```

Comment:
Same question as the other PR. Is end date nullable? If not should we validate start date is before end date?

Reply:
Covered by another validate call on line 87, `validate :start_and_end_dates`. Again there's quite a bit of logic in there so I decided to leave it be